### DISCUSSION TITLE
v2/zh/guide/markdown/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#384] [Announcements]
# v2/zh/guide/markdown/

除了 VuePress 本身新增了一些 Markdown 语法外，vuepress-theme-hope 通过 [vuepress-plugin-md-enhance][md-enhance]，在 Markdown 中启用了更多的语法与新功能。

https://vuepress-theme-hope.gitee.io/v2/zh/guide/markdown/

**Comment by igeekdong** [comment]
test
